EnglishFilter/sections/content_blocker.txt

```diff
@@ -1406,6 +1406,8 @@ m.sports.ru##iframe[src^="http://www.sports.ru/desktop/special/"]
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/41138
 @@||popsci.com/gdpr.html?redirect=
+! https://github.com/AdguardTeam/AdguardFilters/issues/41928
+@@||imasdk.googleapis.com/js/sdkloader/ima3$script,domain=worldsurfleague.com
```

Comment:
why not `imasdk.googleapis.com/js/sdkloader/ima3.js`?

Reply:
unblocked one rule isn't enough, need to additionally unblock similar request.

Reply:
ok